web-console/4.0-designs/global-config/global-config.md

```diff
@@ -0,0 +1,25 @@
+# Global Config
+
+![1](img/D-01B.png)
+
+- Global Config is the third tab of the [Cluster Settings](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/cluster-settings/cluster-settings) page.
+- The tab shows a list view of resources with a search bar in the top right.
+- For first time users, some hint text may be added at the top of the page to better explain Global Configs and link directly to documentation.
+- Users can dismiss the blue hint box at any time.
+
+![2](img/D-01.png)
+
+- An edit action is available at the end of the rows to take the user directly to the YAML tab of the resource detail view. Otherwise, the Overview tab is the default when navigating to the resource.
+- Clicking a resource name will bring the user to the detail view of that resource.
+- Breadcrumbs will appear above the resource name on these pages in order to allow users to navigate back to the global config list view and general Cluster Settings.
```

Comment:
@beanh66 agreed that a breadcrumb needs to be shown, but unsure if what we are showing in the image makes sense.

Reply:
@serenamarie125 What would you expect the breadcrumb to be?